Assets/Scripts/Weapons/BehaviourTypes/BulletMono.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletMono : MonoBehaviour
+{
+    float _damage;
+    public float Damage 
```

Comment:
Why use getters/setters that are almost exactly the same as directly modifying the attributes themselves? Additionally, should bullets be publicly modifiable, or should their damage be set on instantiation and left alone?

Reply:
Yeah no just on instanciation